interfaces/cython/cantera/test/test_onedim.py

```diff
@@ -952,3 +952,28 @@ def test_ion_profile(self):
 
         # Regression test
         self.assertNear(max(self.sim.E), 132.1922, 1e-3)
+
+
+class TestBurnerIonFlame(utilities.CanteraTest):
+    def test_ion_profile(self):
+        reactants = 'CH4:1.0, O2:2.0, N2:7.52'
+        p = ct.one_atm
+        Tburner = 400
+        width = 0.03
+
+        # IdealGasMix object used to compute mixture properties
+        self.gas = ct.Solution('ch4_ion.cti')
+        self.gas.TPX = Tburner, p, reactants
+        self.sim = ct.BurnerIonFlame(self.gas, width=width)
+        self.sim.set_refine_criteria(ratio=4, slope=0.8, curve=1.0)
+        self.sim.burner.mdot = self.gas.density * 0.15
+        self.sim.transport_model = 'Ion'
+
+        # stage one
+        self.sim.solve(loglevel=0, auto=True)
+
+        #stage two
+        self.sim.solve(loglevel=0, stage=2, enable_energy=True)
+
+        # Regression test
+        self.assertNear(max(self.sim.E), 469.7287, 1e-3)
```

Comment:
I'm a little concerned by the fact that check seems to be quite sensitive to small changes in the code or compilation environment. Do you have any idea why, or if there is a output metric that would be better to use for these regression tests? On my computer (Ubuntu 16.04), the test succeeds using G++, but fails with Clang++ or the Intel compiler.

Reply:
I notice that setBounds(c_offset_Y + m_kElectron, -1e-16, 1.0) can affect the test result of IonBurnerFlame, but not IonFreeFlame. Is it possible for me to do travis-ci test by myself?

Reply:
Yes, you can set up an account on Travis and have it build commits on branches pushed to your fork. Or it's also fine to just force push changes to the branch used for this PR and have Travis build from those.